modules/navier_stokes/include/materials/ConservedVarValuesMaterial.h

```diff
@@ -0,0 +1,52 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Material.h"
+
+class SinglePhaseFluidProperties;
+
+class ConservedVarValuesMaterial : public Material
```

Comment:
Otherwise it would conflict with the existing `ConservedVarMaterial` in Pronghorn. Don't make me deal with the name clash again :cry: I think that once this is in we can basically just make `ConservedVarMaterial` a shim of this